### PR TITLE
Polish mini version for new mini(twitter) version - #319

### PR DESCRIPTION
**Pull request by jrgarou**
**Issue**
#319

Some changes and fixes to the  _new_ mini version:
- Short-cut our thumb titles with "..." #298 and same to video title and description
- Polish the controls for twitter #318
- Limit tags not to overflow the paused screen #282
- Modals size and spaces

### CODE REVIEW

File: imports/ui/stylesheets/base/settings.less [hunk 1]
```diff
@@ -35,8 +35,8 @@
 
 @screen-tiny-height: 200px;
 @screen-tiny-width: 400px;
-@screen-mini-height: 320px;
-@screen-mini-width: 570px;
+@screen-mini-height: 280px;
```
Comment (bent0b0x) [inline imports/ui/stylesheets/base/settings.less:38]: Nice, I had just noticed that our embedded height was bigger than the height twitter provides for us. Thanks for taking care of this 👍

Comment (jrgarou) [reply imports/ui/stylesheets/base/settings.less:38]: There's some queries for our breakpoints too, @bent0b0x :
https://github.com/Paratii-Video/paratii-player/blob/dev/imports/ui/stylesheets/base/mixins.less

Comment (bent0b0x) [reply imports/ui/stylesheets/base/settings.less:38]: Great, these are awesome!

**Comment by jrgarou** [comment]
```
1) should navigate from an individual playlist to the playlists page even after the user has navigated directly to a playlist page

1) Page Navigation: playlists page should navigate from an individual playlist to the playlists page even after the user has navigated directly to a playlist page :

Uncaught Promise was rejected with the following reason: timeout
```

**This timeout error sometimes not happen.** 
**In my local when I test `npm run chimp-test`  it got other error:**

` 1) should stay in full-screen mode when a video is paused via the space bar`
```
1) Player: should stay in full-screen mode when a video is paused via the space bar:
Uncaught Error: element not visible
at Object.<anonymous> (tests/helpers.js:42:17)
```

**Pull request merged**